### PR TITLE
Fix null step ID

### PR DESCRIPTION
**Pull request by garyluu**
Apparently there's a few issues with my previous PR (assuming stepDefinition ID exists), this fixes that and adds 2 more tests

https://github.com/garyluu/dockstore-galaxy-interface/runs/592845368 to actually see tests passing this time.

galaxy-workflow-dockstore-example-1:

image

galaxy-workflow-dockstore-example-2:

image

training-materials (good luck trying to read this):
image

### CODE REVIEW

File: src/main/java/org/galaxyproject/dockstore_galaxy_interface/language/GalaxyWorkflowPlugin.java [hunk 1]
```diff
@@ -69,36 +68,43 @@ public String launchInstructions(String trsID) {
     }
 
     @Override
-    public List<RowData> generateToolsTable(String initialPath, String contents, Map<String, Pair<String, GenericFileType>> indexedFiles) {
+    public List<RowData> generateToolsTable(
```
Comment (denis-yuen) [inline src/main/java/org/galaxyproject/dockstore_galaxy_interface/language/GalaxyWorkflowPlugin.java:71]: Reformat for these parameters to be one per line is a bit odd, shouldn't be Dockstore style or Google style.

Comment (garyluu) [reply src/main/java/org/galaxyproject/dockstore_galaxy_interface/language/GalaxyWorkflowPlugin.java:71]: it's from the mvn plugin. i didn't add it

Comment (denis-yuen) [reply src/main/java/org/galaxyproject/dockstore_galaxy_interface/language/GalaxyWorkflowPlugin.java:71]: The mvn-maven-plugin? 
Weird, I assume they formatted their own code and it looks like 
https://github.com/coveooss/fmt-maven-plugin/blob/master/src/main/java/com/coveo/AbstractFMT.java#L224

Comment (garyluu) [reply src/main/java/org/galaxyproject/dockstore_galaxy_interface/language/GalaxyWorkflowPlugin.java:71]: Not sure if they format their own code, but that example seems consistent.  If it's long, make it one per line, else make it one parameter per line.

Comment (denis-yuen) [reply src/main/java/org/galaxyproject/dockstore_galaxy_interface/language/GalaxyWorkflowPlugin.java:71]: :shrug:

**Pull request merged**